src/app/app.component.ts

```diff
@@ -21,6 +21,7 @@ export class OrganizerAppComponent {
 
   @ViewChild(Nav) public nav: Nav;
   public rootPage: any;
+  private disconnectSubscription: any;
```

Comment:
I imported `Subscription` using `import {Subscription} from '@reactivex/rxjs/es6/Subscription.js'`.
Still giving error `Cannot find name 'Subscription'`.
Is there anywhere else where I will need to import the package??